.github/workflows/ci.yml

```diff
@@ -6,7 +6,12 @@ on:
   pull_request:
 
 env:
-  RACK_ENV: ci
+  LAUTH_API_DB_HOST: 127.0.0.1
+  LAUTH_API_DB_DATABASE: test
+  LAUTH_API_DB_USER: user
+  LAUTH_API_DB_PASSWORD: password
+  LAUTH_CLI_API_URL: "http://0.0.0.0:9292"
+  LAUTH_CLI_API_DB_URL: "mysql2://user:password@127.0.0.1/test?encoding=utf8mb4"
```

Comment:
I think these two are strictly for testing purposes. If that's the case, I'd indicate that in the name and shorten them up. Maybe `LAUTH_TEST_API_URL`, `LAUTH_TEST_DB_URL`?

Reply:
Yes, agreed.